src/controllers/collections.js

```diff
@@ -0,0 +1,9 @@
+import { fetchCollections } from '../services/collections.js';
+
+export const getCollections = async (req, res) => {
+  try {
+    res.json(await fetchCollections());
+  } catch (error) {
+    res.status(error.response?.status || 500).json({ error: `${error.message} (${error.code})` });
```

Comment:
Not exactly because errorsMiddleware can't catch asynchronous exceptions (this will be done in ExpressJS 5). But as I recognise that this can be confusing, I'm going to use this very small [express-async-errors](https://github.com/davidbanham/express-async-errors/) package to act as Express 5 will and to allow catches to be removed.